spec/support/build_test_app.rb

```diff
@@ -36,9 +36,14 @@ def ensure_scaffold
         scaffold_dir.rmtree if scaffold_dir.exist?
         TEST_DIR.mkpath
 
-        shell_run "rails new #{scaffold_dir} #{rails_new_options}"
-
         require 'rails'
+
+        if Rails::VERSION::MAJOR > 5
+          shell_run "rails new #{scaffold_dir} #{rails_new_options} --skip-javascript"
```

Comment:
why did we add `--skip-javascript`?

Reply:
Good question. We either do this or we have to require `webpacker` as a gem dependency for rails 6. Since we aren't going to be building new rails apps with frontends in rails I think this is fine as coverage.

Reply:
Fair enough ✅